### DISCUSSION TITLE
Issue with map display showing up

### DISCUSSION THREAD

**Discussion by SanPatricio72** [Khopa/dcs_liberation#1549] [Q&A] [answered]
So I've been using liberation for awhile but yesterday when I went to launch 4.1.1 the central map display was not visible.  Just a blank grey area where the map should be.  I tried creating a new campaign on a different map and rolling back to 4.1.0 and 4.0 and nothing fixed it.  I launches fine, but the map is not visible.  I just tried again and noticed that when I don't launch as Administrator then the map loads fine but as soon as a shift over to launching as Admin it vanishes.  The only change that I've made to my PC since it last worked was to add a new account on the machine but I'm using my old (Admin) account.  Any suggestions would be appreciated.  Thanks!

**Comment by Mustang-25** [comment]
I'm not sure what older version of Liberation you were running but the map is different from older versions of Liberation in that it's now web-based. I would think that running as admin would work. Do you have an anti-virus/network monitor that's blocking access while running as admin?

**Reply by SanPatricio72** [reply]
I was using 4.1.1 and it worked fine until last week.  I tried disabling my firewall and anti-virus but no dice.  It's odd cos it loads the map so long as I don't run ad Admin.

**Reply by Mustang-25** [reply, marked as answer]
Not sure why it wouldn't work as admin and not the other way around... But I guess if you can run Liberation without running as admin then just keep doing that.
